app/endpoints/core.py

```diff
@@ -127,3 +133,103 @@ async def get_style_file(
 )
 async def get_favicon():
     return FileResponse("assets/images/favicon.ico")
+
+
+@router.get(
+    "/module_visibility/",
```

Comment:
It is usually recommended to use hyphen as word separator in RestAPI url, this is also the convention we used for other Hyperion's endpoints